install_config/sssd_for_ldap_failover.adoc

```diff
@@ -0,0 +1,421 @@
+= Setting up SSSD for LDAP Failover
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+{product-title} provides an
+xref:../install_config/configuring_authentication.adoc#LDAPPasswordIdentityProvider[authentication
+provider] for use with Lightweight Directory Access Protocol (LDAP) setups, but
+it can only connect to a single LDAP server. This can be problematic if that
+LDAP server becomes unavailable. System Security Services Daemon (SSSD) can be
+used to solve the issue.
+
+Originally designed to manage local and remote authentication to the host
+operating system, SSSD can now be configured to provide identity,
+authentication, and authorization services to web services like {product-title}.
+SSSD provides advantages over the built-in LDAP provider, including the ability
+to connect to any number of failover LDAP servers, as well as the ability to
+cache authentication attempts in case it can no longer reach any of those
+servers.
+
+The setup for this configuration is advanced and requires a separate
+authentication server (also called an *authenticating proxy*) that
+{product-title} will communicate with. This topic describes how to do this setup
+on a dedicated physical or virtual machine (VM), but the concepts are also
+applicable to a setup in a container.
+
+[[sssd-prerequisites-for-authenticating-proxy-setup]]
+== Prerequisites for Authenticating Proxy Setup
+
+Before starting setup, you need to know the following information about your
+LDAP server.
+
+* Whether the directory server is powered by
+http://www.freeipa.org/page/Main_Page[FreeIPA], Active Directory, or another
+LDAP solution.
+
+* The Uniform Resource Identifier (URI) for the LDAP server (for example,
+ldap.example.com).
+
+* The location of the CA certificate for the LDAP server.
+
+* Whether the LDAP server corresponds to RFC 2307 or RFC2307bis for user groups.
+
+Prepare the VMs:
+
+* *_proxy.example.com_*: A VM to use as the authenticating proxy. This machine must
+have at least SSSD 1.12.0 available, which means a fairly recent operating
+system. Throughout this topic, we will use a Red Hat Enterprise Linux 7.2
```

Comment:
Avoid future tense and don't say "we": This topic uses a Red Hat Enterprise Linux 7.2 server for its examples.